buildkite/bazelci.py

```diff
@@ -702,6 +704,12 @@ def print_bazel_version_info(bazel_binary):
     execute_command([bazel_binary, "--nomaster_bazelrc", "--bazelrc=/dev/null", "info"])
 
 
+def print_environment_variables_info():
+    print_collapsed_group(":information_source: Environment Variables")
+    for key, value in os.environ.items():
```

Comment:
Also, please sort this dict.

Reply:
Just tested, this dict is already sorted.